models/pp_cond_exp_mc_urbanczik.h

```diff
@@ -107,82 +107,140 @@ class pp_cond_exp_mc_urbanczik_parameters
+each compartment.
 
 Synapses, including those for injection external currents, are addressed through
 the receptor types given in the receptor_types entry of the state dictionary.
 Note that in contrast to the single-compartment models, all
 synaptic weights must be positive numbers!
 
-
-Parameters:
+.. _multicompartment-models:
+
+Multicompartment models and synaptic delays
++++++++++++++++++++++++++++++++++++++++++++
+
+Note that in case of multicompartment models which represent the dendrite
+explicitly, the interpretation of the synaptic delay in NEST requires a careful
+consideration. In NEST, the delay is at least one simulation time step and is
+assumed to be located entirely at the postsynaptic side. For point neurons, it
+represents the time it takes for an incoming spike to travel along the
+postsynaptic dendrite before it reaches the soma, see :ref:`panel a)
+<fig-multicompartment>`. Conversely, if the synaptic weight depends on the
+state of the postsynaptic neuron, the delay also represents the time the
+information on the state propagates back through the dendrite to the synapse.
+
+For multicompartment models, this amounts to positioning the delay directly
+behind the incoming synapse, i.e., before the first dendritic compartement on
+the postsynaptic side, see :ref:`panel b) <fig-multicompartment>`. Therefore,
+the delay specified in the synapse model does *not* account for any delay that
+might be associated with information traveling through the explicitly modeled
+dendritic compartments.
+
+In case of the Urbanczik synapse, the change of the synaptic weight is driven by
+an error signal which is the difference between the firing rate of the soma
+(derived from the somatic spike train :math:`S_{post}`) and the
+dendritic prediction of it (derived from the dendritic membrane potential
+:math:`V`). The original publication [1]_ does not assume any delay in the
+interaction between the soma and the dendritic compartment. Therefore, we
+compute the error signal from the firing rate and the dendritic prediction at
+equal time points. Due to the synaptic delay :math:`d`, the synapse combines a
+delayed version of the error signal with the presynaptic spike train
+(:math:`S_{pre}`), see :ref:`panel c) <fig-multicompartment>`.
+
+.. _fig-multicompartment::
+
+.. figure:: ../static/img/multicompartment.png
+   :width: 75 %
+
+   a) Two point neurons (red circles *pre* and *post*) connected via a synapse.
+   In NEST, the delay is entirely on the postsynaptic side and in case of point
+   neurons it is interpeted as the dendritic delay. b) Two two-compartment
+   neuron models composed of a somatic (green) and a dendritic (blue)
+   compartment. The soma of the presynaptic neuron is connected to the dendrite
+   of the postsynaptic neuron. The synaptic delay is located behind the synapse
+   and before the dendrite. c) Time trace of the State variables that enter the
```

Comment:
I am a bit confused by what c) is trying to say. 
Can you be specific as to which variables you are referring to? 
What are postsynaptic quantities - quantity is very vague term.
Is the sentence implying that the dendritic membrane potential and the somatic spike train are the aforementioned postsynaptic quantities? (If so, there should be a colon : rather than semicolon.)
How is the presynaptic spike train _combined with_ a delayed version of postsynaptic quantities? Combined how?

Reply:
@jessica-mitchell The variables I am referring to are the presynaptic spike train `S_pre`, the postsynaptic spike train `S_post`, and the dendritic membrane potential `V`. Therefore, yes, the dendritic membrane potential and the somatic spike train are the aforementioned postsynaptic quantities and the semicolon should be replaced by a colon.
Regarding the question how these quantities are combined, I do not think that it should be explained in more detail here since it is done properly in reference [1] which provides all formulas and derivations.

Reply:
ok, sounds good